### PR TITLE
Deploy the 01/Sept/2017 rebase - issue 222

### PR DESCRIPTION
**Pull request by tonikitoo**
Issue #222 

TBR=msisov

### CODE REVIEW

**Comment by tonikitoo** [comment]
TBR=msisov

File: services/ui/ws/window_tree.cc [hunk 1]
```diff
@@ -910,7 +910,7 @@ void WindowTree::ProcessWindowBoundsChanged(
   }
 
   client()->OnWindowBoundsChanged(ClientWindowIdToTransportId(client_window_id),
-                                  old_bounds, new_bounds, local_surface_id);
+                                  old_bounds, newest_bounds, local_surface_id);
```
Comment (msisov) [inline services/ui/ws/window_tree.cc:913]: Excuse me, but how does it relate to the commit you've mentioned in the commit message?

Comment (tonikitoo) [reply services/ui/ws/window_tree.cc:913]: I think you are right. this should be related to "fixup! Handle OnBoundsChange in external mode.".

Comment (msisov) [reply services/ui/ws/window_tree.cc:913]: I'll fix it during rebasing if you don't mind.

**Pull request merged**